examples/async-streaming/README.md

```diff
@@ -0,0 +1,83 @@
+# Asynchronous Streaming
+
+## Overview
+The primary goal of this example is to demonstrate how to implement an asynchronous gRPC call in Ballerina. Here, the RPC call happens in one strand (i.e., a lightweight thread in Ballerina) while the other phone call processing happens in another strand simultaneously. This is the Ballerina implementation of [this asynchronous streaming Python example](https://github.com/grpc/grpc/blob/v1.43.2/examples/python/async_streaming/phone.proto).
+
+## Implementation
+This example contains a bidirectional RPC. In Ballerina, the invocation of a bidirectional RPC (or even in a client streaming RPC) returns a streaming client, which can be used to receive or send messages.
+```ballerina
+StreamCallStreamingClient streamingClient = check ep->StreamCall();
+// ...
+
+// Send messages using the streaming client
+check streamingClient->sendStreamCallRequest({phone_number: phoneNumber});
+
+// Complete the message sending
+check streamingClient->complete();
+
+// Receive the messages from the server
+StreamCallResponse? response = check streamCall->receiveStreamCallResponse();
```

Comment:
```suggestion
StreamCallResponse? response = check streamingClient->receiveStreamCallResponse();
```

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-grpc/pull/706/commits/8730cd8cff2acbeef602532011988b6e4fd0fab5